runtime/runtime/tests/test_async_calls.rs

```diff
@@ -21,7 +27,7 @@ fn test_simple_func_call() {
         vec![Action::FunctionCall(FunctionCallAction {
             method_name: "sum_n".to_string(),
             args: 10u64.to_le_bytes().to_vec(),
-            gas: 1_000_000_000,
+            gas: 10u64.pow(19),
```

Comment:
This is intentionally close to `u64::MAX` because I did not want to figure out how much gas to attach for all our tests. I haven't profiled some of our common contracts in terms of gas usage yet, though. I think we should do it first before we decide whether we need to change `u64->u128`. Created https://github.com/nearprotocol/nearcore/issues/1829